.github/workflows/_build.yaml

```diff
@@ -140,3 +140,14 @@ jobs:
           ${{ env.PACKAGE_PATH }}/bin/bashparser
```

Comment:
After our discussion, we decided to remove these artifacts as they will be part of the wheel anyway?